x-pack/plugins/siem/public/components/page/hosts/hosts_table/columns.tsx

```diff
@@ -40,8 +40,8 @@ export const getHostsColumns = (
     hideForMobile: false,
     sortable: true,
     render: hostName => {
-      if (hostName != null) {
-        const id = escapeDataProviderId(`hosts-table-hostName-${hostName}`);
+      if (hostName != null && hostName.length > 0) {
```

Comment:
`hostName` is a string according to its type but you're using it like an `Array`. I would fix the typing if they're incorrect and then use it as an array.

Reply:
To my surprise too, we had an issue with type sharing between detail and hosts query and now it is an array of string